shared/js/loadApp.js

```diff
@@ -56,7 +58,13 @@ module.exports = function (callback) {
   }
 
   if (window.dashboard && dashboard.project) {
-    promise = promise.then(dashboard.project.load);
+    promise = promise.then(dashboard.project.load)
+    .then(function () {
+      if (dashboard.project.hideBecauseAbusive()) {
+        renderAbusive();
+        return $.Deferred().reject();
```

Comment:
Ah I'm mistaken.  It looks like this does stop the promise chain:
```javascript
var promise = new $.Deferred();
promise.then(function () { console.log('a') }).then(function () { return $.Deferred().reject(); }).then(function () { console.log('b') });
promise.resolve();
```
Logs "a" but not "b".